### DISCUSSION TITLE
Código de texto visible

### DISCUSSION THREAD

**Discussion by EpicPoyai** [FakeFishGames/Barotrauma#13972] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

El código es visible, no deja ver cuales son los beneficios que da la habilidad.

![Captura de pantalla 2024-05-17 183450](https://github.com/FakeFishGames/Barotrauma/assets/170150353/a7bf353d-f521-43b0-8918-713d27db6c56)

### Reproduction steps

1. Simplemente poniendo el trabajo de "Médico" en alguna partida y viendo sus habilidades.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by kaedepon** [comment]
This report was discussed in https://github.com/FakeFishGames/Barotrauma/issues/13817
The issue is closed and the development team says it will be fixed in the next hotfix (v1.4.6.0)
https://github.com/FakeFishGames/Barotrauma/issues/13817#issuecomment-2082229637
```
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/14b11e86754a23dc96f2d1bd687dc8a1425fe3d3, hopefully coming up in the next hotfix!
```
Hotfix v1.4.6.0
```
- Removed duplicate localization lines that caused old versions of text to show up in some places.
```
Unfortunately, It removed localization lines that latest versions of text I think.


## War stories / English
![warstory_english](https://github.com/FakeFishGames/Barotrauma/assets/47914478/27331a60-63b7-45c8-aa7d-fd372ef2a294)
## War stories / Espanol
![warstory_espanol](https://github.com/FakeFishGames/Barotrauma/assets/47914478/2f79559b-5d21-4dc4-a3ce-2858affafb5c)
## War stories / Japanese
![warstory_japanese](https://github.com/FakeFishGames/Barotrauma/assets/47914478/9963e938-5235-42b7-9f8f-c96a80c1b268)

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13839
